dle/search/templates/search/elastic/search.html

```diff
@@ -43,7 +43,7 @@ <h3>Drug Label Generic Name</h3>
     const sk = new Searchkit({
       connection: {
         host: "https://localhost:9200", // TODO remove hardcoding for deployment
-        apiKey: "aF9xVEVJY0JLbDFoeUxRVlFzUzY6SEdEbng2dy1SMC1oZktieUlZTFczQQ==" // TODO remove hardcoding for deployment
+        apiKey: "LUQ2Q0VZY0I3SmJyS1lSZjdiREI6dHZHUWRhMEZSbXVWa3R0RlhpTmZ1Zw==" // TODO remove hardcoding for deployment
```

Comment:
We need to use Django template interpolation or a relative URL to point to the proxy URL in the `api` app, not the Elasticsearch port. No apiKey needed for now as it is proxed from the backend which has a direct connection. We should add back an API key later for users who are querying the (to be built) API from external IPs but that's an auth method we would implement, probably not through Elastic directly as then we'd need to create a bunch of Elastic users and keys which seems less than ideal.